tools/azure-sdk-tools/devtools_testutils/proxy_testcase.py

```diff
@@ -208,8 +216,9 @@ def combined_call(*args, **kwargs):
                 test_output = test_func(*args, **trimmed_kwargs)
         except ResourceNotFoundError as error:
             error_body = ContentDecodePolicy.deserialize_from_http_generics(error.response)
-            error_with_message = ResourceNotFoundError(message=error_body["Message"], response=error.response)
-            raise error_with_message
+            message = error_body.get("message") or error_body.get("Message")
+            error_with_message = ResourceNotFoundError(message=message, response=error.response)
+            six.raise_from(error_with_message, error)
```

Comment:
What does `six.raise_from` do here? Just an easy way to say 

> raise an exception that matches this other one, but provide a different body.

?

Reply:
This will raise the `error_with_message` that we parsed from the server response, but will raise it in the context of the original exception we caught, so we'll be able to see the full exception chain

Reply:
That's what I had guessed, thanks for the clarification! Great change and my sign off stands 👍